ecosystem/sep-0006.md

```diff
@@ -69,6 +73,58 @@ Access-Control-Allow-Origin: *
 
 This protocol involves the transfer of value, and so HTTPS is required for all endpoints for security.  Wallets and anchors should refuse to interact with any insecure HTTP endpoints.
 
+## Asset Exchanges
+
+This protocol was originally designed to provide conversions between off-chain and on-chain equivalent assets – for instance `BRL <> BRLT`, `USD <> USDC`, `NGN <> NGNT`, etc. – but it also describes how to make exchanges between off-chain and on-chain non-equivalent assets like `BRL <> USDC`.
+
+To enable the conversion of non-equivalent assets, the Anchor needs to implement the [SEP-38 Anchor RFQ API](sep-0038.md), which enables the Anchor to provide quotes to the Wallet, as well as the endpoints [`GET /deposit-exchange`](#deposit-exchange) and [`GET /withdraw-exchange`](#withdraw-exchange).
+
+### Asset Conversions Using the Market Rate
+
+To convert at the market rate, clients first need to get an [indicative quote](https://www.investopedia.com/terms/i/indicativequote.asp) with [SEP-38] and then proceed with the conversion flow. With market rate conversions there won't be a firm guarantee of the final price to be paid for the asset. Here is how this kind of conversion should happen in deposits and withdrawals for an asset pair like `BRL <> USDC`:
+
+**Deposit:**
+
+Considering the user will be making a bank deposit of fiat `BRL` in exchange for `USDC` in the Stellar network:
+
+1. The Wallet checks the assets' market conversion rate in [`SEP-38 GET /price`](sep-0038.md#get-price) and shows the estimated price to the user.
+1. The Wallet initiates a [`GET /deposit-exchange`](#deposit-exchange) using the parameters `source_asset` and `destination_asset`.
+1. The user proceeds with the off-chain deposit, sending `BRL` to the Anchor's bank account.
+1. Upon receiving the off-chain deposit funds, the Anchor converts it to `USDC` using the market rate and sends the resulting amount to the user's Stellar account.
+
+**Withdraw:**
+
+Considering the user will be making a Stellar payment of `USDC` in exchange for withdrawing `BRL` in the bank account:
+
+1. The Wallet checks the assets' conversion rate in [`SEP-38 GET /price`](sep-0038.md#get-price) and shows the estimated price to the user.
+1. The Wallet initiates a [`GET /withdraw-exchange`](#withdraw-exchange) using the parameters `source_asset` and `destination_asset`.
+1. The user proceeds with the on-chain payment, sending `USDC` to the Anchor's Stellar account.
+1. Upon receiving the on-chain withdraw funds, the Anchor converts it to `BRL` using the market rate and sends the resulting amount to the user's bank account.
+
+### Asset Conversions Using a Firm Quote
+
+With firm quotes, users can have a guarantee of the asset price they are going to pay **if they submit the payment before the quote expiration**. Here is how the conversion should happen in deposits and withdrawals for an asset pair like `USDC <> BRL`:
```

Comment:
🔡 Also, avoid emphasising text in SEPs that aren't headings. All the text in SEPs is important.
```suggestion
With firm quotes, users are guaranteed an asset price they will pay if they submit the payment before the quote expiration. Here is how the conversion should happen in deposits and withdrawals for an asset pair like `USDC <> BRL`:
```

Reply:
https://github.com/stellar/stellar-protocol/pull/973/commits/7283bfcc3c38b3d24185becc32543a4dc83ca4de